build.js

```diff
@@ -131,6 +131,10 @@ if (useGoma) {
 const gn = currentPlatform() == "windows" ? "gn.bat" : "gn";
 spawnChecked(gn, ["gen", outdir], { stdio: "inherit" });
 
+console.log(`Linting replay js blobs...`);
+spawnChecked("npm", ["ci", `${__dirname}/replay_build_scripts`], { stdio: "inherit" });
+spawnChecked("node", [`${__dirname}/replay_build_scripts/lint.mjs`, `${__dirname}/third_party/blink/renderer/bindings/core/v8/record_replay_interface.cc`], { stdio: "inherit" });
+
```

Comment:
a previous commit had this happening as a preflight check in CI, which I kinda like because if it fails (and it fails relatively fast), we don't have a lot of jobs taking up space in the buildkite queues.  But then we're doing it twice: once in the buildkite pipeline, and then again for each platform build.

If we'd rather, I can add it back to buildkite, and we can either live with running lint multiple times, or the our CI build step could always pass an additional flag to build.js to skip the lint.

Reply:
I added it back in as a preflight step and added the disabling during build.js (but only on buildkite - running `node build` locally will run lint.)